Source/FieldSolver/SpectralSolver/SpectralAlgorithms/SpectralBaseAlgorithm.cpp

```diff
@@ -107,5 +103,6 @@ SpectralBaseAlgorithm::ComputeSpectralDivE (
     }
 
     // Backward Fourier transform
-    field_data.BackwardTransform(lev, divE, Idx.divE, 0, fill_guards);
+    const amrex::IntVect& fill_guards = amrex::IntVect(0);
+    field_data.BackwardTransform(lev, divE, Idx.divE, fill_guards, 0);
```

Comment:
Here I'm setting `fill_guards` so that the guard cells are **not** filled for the spectral div(E) diagnostics, because I feel like it isn't really necessary to do so. Please let me know if you think they should be filled instead.

Reply:
I agree.